Material.Blazor.Website/Pages/Dialog.razor

```diff
@@ -203,41 +203,44 @@
                   ScrimClickAction=""
                   Title="Logon Dialog">
             <Body>
-                <EditForm Model="@UserLogon"
-                          OnInvalidSubmit="@Dialog4Invalid"
-                          OnValidSubmit="@Dialog4Submitted">
-                    <DataAnnotationsValidator />
-
-                    <div class="mdc-layout-grid__inner">
-                        <div class="mdc-layout-grid__cell--span-12">
-                            <MBTextField @bind-Value="@UserLogon.UserID"
-                                         Label="User ID"
-                                         style="width: 100%; margin-top: 6px;"
-                                         TextInputStyle="@MBTextInputStyle.Outlined"
-                                         TrailingIcon="person"
-                                         ValidationMessageFor="@(() => UserLogon.UserID)" />
+                @if (UserLogon != null)
```

Comment:
This is now necessary, because dialogs are added to the DOM sooner.

TODO: make a remark in the Changelog, because this might affect some users.

Reply:
TODO: test whether this change is really still necessary